apps/sentry-client-desktop/src/features/keys/HasKeys.tsx

```diff
@@ -245,7 +245,7 @@ export function HasKeys({combinedOwners, combinedLicensesMap, statusMap, isWalle
 							}
 							setSelectedValue={setSelectedWallet}
 							getDropdownItems={getDropdownItems}
-							extraClasses={{dropdown: "max-w-[456px]"}}
+							extraClasses={{dropdown: "max-w-[456px]", dropdownOptions: "w-[456px]"}}
```

Comment:
@XDapps same as the comment above

Reply:
This one seems to be correct 

![image](https://github.com/user-attachments/assets/034da543-9ac6-4142-a31e-89d7700b3db1)